src/GraphQL/Types/ResolveFieldContext.cs

```diff
@@ -79,42 +79,22 @@ public ResolveFieldContext(ResolveFieldContext context)
 
         public TType GetArgument<TType>(string name, TType defaultValue = default)
         {
-            return (TType)GetArgument(typeof(TType), name, defaultValue);
+            return ResolveFieldContextExtensions.GetArgument(this, name, defaultValue);
```

Comment:
If three methods have become extension methods and backward compatibility has been already broken, then it makes no sense to leave these three methods here. Just remove.

Reply:
The methods are there to increase backwards compatibility.  Additional changes can ensure 100% compatibility.  However, I would agree to remove them, as 3.0 has not been released yet.

Reply:
> The methods are there to increase backwards compatibility. 

Prior to that, there were dozens of changes that completely killed backward compatibility.

Reply:
Done